syntax/ast.mjs

```diff
@@ -147,11 +147,12 @@ export class VariantExpression extends Expression {
 }
 
 export class CallExpression extends Expression {
-    constructor(callee, args = []) {
+    constructor(callee, args = [], kwargs = []) {
         super();
         this.type = "CallExpression";
         this.callee = callee;
         this.arguments = args;
+        this.keywords = kwargs;
```

Comment:
The runtime has to separate positional and named arguments anyways, so we might just as well do it in the AST. I called the field `keywords` following how Python calls them. The arguments, however, are called _named arguments_, so perhaps there's some better wording here?

Reply:
I wouldn't introduce `keyword` in fluent. I'd make positional vs named explicit, as in `positional_arguments` and `keyword_arguments`. If that's too long, maybe `_args`, or just no suffix, and comment `// arguments`.

Reply:
What do you mean by `just no suffix, and comment // arguments.`? I also considered `options` for the named arguments, since they are mostly intended to be used as such.

Reply:
No suffix meant to say
```
// arguments
positional = pos_args
named = named_args
```

Reply:
Ah, OK! Yeah, I think I'll go for this one.

Reply:
I'm going to land this PR with `positional` and `named`. I filed #124 to discuss renaming `named` to `options`.